consumer.go

```diff
@@ -229,7 +229,7 @@ func (c *Consumer) Offset(topic string, partitionID int32) (int64, error) {
 func (c *Consumer) Ack(msg *sarama.ConsumerMessage) {
 	tp := topicPartition{msg.Topic, msg.Partition}
 	c.aLock.Lock()
-	if msg.Offset > c.acked[tp] {
```

Comment:
Wouldn't that commit all offsets all the time? If the last committed offset was 11, and msg.Offset is also 11 it will be recommitted.